pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -241,6 +241,10 @@ func TestReconcile(t *testing.T) {
 	machineAlreadyDeleted := maotesting.NewMachine("machineAlreadyDeleted", nodeAlreadyDeleted.Name)
 	machineAlreadyDeleted.SetDeletionTimestamp(&metav1.Time{Time: time.Now()})
 
+	// machine with external remediation annotation, but with now healthy node
+	machineHealthyWithExtRemAnnotation := maotesting.NewMachine("healthyWithExtRemAnn", nodeHealthy.Name)
+	machineHealthyWithExtRemAnnotation.Annotations[machineExternalAnnotationKey] = ""
```

Comment:
(NTH) I realize that the value of the annotation isn't important (only it's existence), however I still think it'll be better to use in at least one of the cases none-empty string.
That is in the off chance that latter on the production code will actually regard the value.

Reply:
we are in full control of how this annotation is used, and it will be replaced by the external remediation feature soonish anyway... so I think this is fine